stdlib/public/core/StringGraphemeBreaking.swift

```diff
@@ -13,8 +13,8 @@
 import SwiftShims
 
 /// CR and LF are common special cases in grapheme breaking logic
-private var _CR: UInt8 { return 0x0d }
-private var _LF: UInt8 { return 0x0a }
+@usableFromInline internal var _CR: UInt8 { return 0x0d }
+@usableFromInline internal var _LF: UInt8 { return 0x0a }
```

Comment:
Surely these could be inlinable?

Reply:
Probably yes! I was trying to make the absolute minimum change to get it building, then refactor once it worked. But you're right, that might totally invalidate my attempt to benchmark.